src/SmartComponents/DriftTable/AddSystem.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+
+export class AddSystem extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <Button
+                variant='primary'
+                onClick={ this.props.getAddSysModal }>
```

Comment:
This is a bit nitpicky, but I tend to prefer consistency when it comes to shortening names. I find this helps with developer predictability and readability. In this case, since `System` is used in most places I would prefer `getAddSystemModal`.